src/lib.rs

```diff
@@ -3,7 +3,7 @@
 //! [ratatui](https://github.com/ratatui-org/ratatui) is a library that is all about cooking up terminal user
 //! interfaces (TUIs).
 //!
-//! ![Demo](https://vhs.charm.sh/vhs-tF0QbuPbtHgUeG0sTVgFr.gif)
+//! ![Demo](https://raw.githubusercontent.com/ratatui-org/ratatui/2ec571113f081373f59d206c1fa51b26c761f81f/examples/demo2.gif)
```

Comment:
Correct me if I'm wrong, but didn't we have a separate branch for handling the GIFs? Should we also use that branch here as well?

Reply:
You're not wrong. This link is is a permalink to the specific version of:
https://github.com/ratatui-org/ratatui/blob/images/examples/demo2.gif